frontend/src/responseQuizView/responseQuizView.js

```diff
@@ -0,0 +1,343 @@
+                                        }
+                                    }
+
+                                    if (!this.state.graded && this.state.question === qNumber) {
+                                        qStatus = 'Current';
+                                    }
+
+                                    return (<>
+                                        <button
+                                            type="button"
+                                            value={key + 1}
+                                            key={key + 1}
+                                            className="pb-question"
+                                            onClick={this.onProgressBarClick}
+                                        >
+                                            Question #{qNumber}
+                                        </button>
+                                        <p className="pb-status"><i>{qStatus}</i></p>
+                                    </>);
+                                })
+                            }
+                        </div>
+                        <div className="col-9 shaded-box">
+                            <h3 className="question-title">Question #{this.state.question}</h3>
+                            <p className="question-primary-text">
+                                Respond to the prompt below:
+                            </p>
+                            <br />
+                            <p className="question-secondary-text">
+                                {this.state.data[this.state.question - 1].question}
+                            </p>
+                            <br />
+                            {(this.state.graded)
+                                ? <div>
+                                    <p>
+                                        <b className="results-header">Your Response</b>
+                                        <br />
+                                        <p className="results-text">
+                                            Your answer: {(Object.prototype.hasOwnProperty.call(
+                                                this.state.userAnswers,
+                                                this.state.question,
+                                            ))
+                                                ? <>{this.state.userAnswers[this.state.question]}</>
+                                                : <i>Unanswered</i>}
+                                        </p>
+                                        <p className="results-text">
+                                            Correct answer: {
+                                                this.state.data[this.state.question - 1].answer
+                                            }
+                                        </p>
+                                    </p>
+                                    <b className="results-header">Machine-Generated Feedback</b>
```

Comment:
This `<b>` tag looks like it could more semantically be rendered as a header tag of some kind. The [usage notes for the `<b>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b#usage_notes) warn against impulse to use `<b>` in place of header tag.